src/Utdanning/Seksjon.elm

```diff
@@ -1337,33 +1363,49 @@ modelTilBrukerInput model =
                     BrukerInput.knapper Flytende
-                        [ Knapp.knapp BrukerVilRegistrereUtdanning "Ja, jeg har utdanning"
-                        , Knapp.knapp (GåTilArbeidserfaring AnnenAvslutning) "Nei, jeg har ikke utdanning"
+                        [ Knapp.knapp (BrukerVilRegistrereUtdanning AlleNivåer) "Ja, jeg har utdanning"
+                        , Knapp.knapp SvarerNeiTilUtdanning "Nei, jeg har ikke utdanning"
                         ]
 
                 else
                     BrukerInput.knapper Flytende
-                        [ Knapp.knapp BrukerVilRegistrereUtdanning "Ja, legg til en utdanning"
+                        [ Knapp.knapp (BrukerVilRegistrereUtdanning AlleNivåer) "Ja, legg til en utdanning"
                         , Knapp.knapp (GåTilArbeidserfaring AnnenAvslutning) "Nei, jeg er ferdig"
                         , Knapp.knapp BrukerVilRedigereUtdanning "Nei, jeg vil endre det jeg har lagt inn"
                         ]
 
+            BekreftHarIkkeUtdanning ->
+                BrukerInput.knapper Flytende
+                    [ Knapp.knapp (BrukerVilRegistrereUtdanning GrunnskoleVideregående) "Ja, det vil jeg"
+                    , Knapp.knapp (GåTilArbeidserfaring AnnenAvslutning) "Nei, det vil jeg ikke"
+                    ]
+
             VelgEnUtdanningÅRedigere ->
                 BrukerInput.knapper Kolonne
                     (lagUtdanningKnapper model.utdanningListe)
 
-            RegistrerNivå ->
+            RegistrerNivå nivåValg ->
                 BrukerInput.knapper Kolonne
-                    [ velgNivåKnapp Grunnskole
-                    , velgNivåKnapp VideregåendeYrkesskole
-                    , velgNivåKnapp Fagskole
-                    , velgNivåKnapp Folkehøyskole
-                    , velgNivåKnapp HøyereUtdanning1til4
-                    , velgNivåKnapp HøyereUtdanning4pluss
-                    , velgNivåKnapp Doktorgrad
-                    , Knapp.knapp BrukerVilAvbryteRegistreringen "Avbryt"
-                        |> Knapp.withType Flat
-                    ]
+                    ([ [ velgNivåKnapp Grunnskole
+                       , velgNivåKnapp VideregåendeYrkesskole
+                       ]
+                     , case nivåValg of
+                        AlleNivåer ->
+                            [ velgNivåKnapp Fagskole
+                            , velgNivåKnapp Folkehøyskole
+                            , velgNivåKnapp HøyereUtdanning1til4
+                            , velgNivåKnapp HøyereUtdanning4pluss
+                            , velgNivåKnapp Doktorgrad
+                            ]
+
+                        GrunnskoleVideregående ->
+                            []
+                     , [ Knapp.knapp BrukerVilAvbryteRegistreringen "Avbryt"
+                            |> Knapp.withType Flat
+                       ]
+                     ]
+                        |> List.concat
+                    )
```

Comment:
Tenker at det kanskje blir enklere å se hva som skjer her hvis du bare har en case `case nivåValg of` øverst, og så returnerer en liste for `AlleNivåer` og en annen liste for `GrunnskoleVideregående`, selv om det blir litt duplisering av knapper.